packages/ibm-products-styles/src/components/UserAvatar/_user-avatar.scss

```diff
@@ -64,6 +65,10 @@ $sizes: (
   }
 }
 
+.#{$block-class}__tooltip button {
+  pointer-events: none;
+}
+
```

Comment:
This actually prevents the tooltip from showing up all on hover.

I think it’s better to leave this as is than to add this. You can compare this to similar components like the [read-only `Tag`](https://react.carbondesignsystem.com/?path=/story/components-tag--read-only) from Carbon. These may be more specific to the browser defaults, which is okay.